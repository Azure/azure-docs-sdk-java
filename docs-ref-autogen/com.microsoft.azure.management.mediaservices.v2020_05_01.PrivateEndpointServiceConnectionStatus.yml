### YamlMime:JavaType
uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus"
fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus"
name: "PrivateEndpointServiceConnectionStatus"
nameWithType: "PrivateEndpointServiceConnectionStatus"
summary: "Defines values for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.microsoft.rest.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedClassMethods:
- classRef: "<xref href=\"com.microsoft.rest.ExpandableStringEnum?alt=com.microsoft.rest.ExpandableStringEnum&text=ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)?alt=com.microsoft.rest.ExpandableStringEnum.<T>fromString&text=<T>fromString\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)?alt=com.microsoft.rest.ExpandableStringEnum.<T>values&text=<T>values\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)?alt=com.microsoft.rest.ExpandableStringEnum.equals&text=equals\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.microsoft.rest.ExpandableStringEnum.hashCode()?alt=com.microsoft.rest.ExpandableStringEnum.hashCode&text=hashCode\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.microsoft.rest.ExpandableStringEnum.toString()?alt=com.microsoft.rest.ExpandableStringEnum.toString&text=toString\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)?alt=com.microsoft.rest.ExpandableStringEnum.withNameValue&text=withNameValue\" data-throw-if-not-resolved=\"False\" />"
- classRef: "java.lang.<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html\">Object</a>"
  methodsRef:
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#clone--\">clone</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#finalize--\">finalize</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#getClass--\">getClass</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notify--\">notify</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notifyAll--\">notifyAll</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait--\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-int-\">wait</a>"
syntax: "public final class **PrivateEndpointServiceConnectionStatus**</br> extends <xref href=\"com.microsoft.rest.ExpandableStringEnum?alt=com.microsoft.rest.ExpandableStringEnum&text=ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />&lt;<xref href=\"com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus?alt=com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />&gt;"
constructors:
- uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  name: "PrivateEndpointServiceConnectionStatus()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  syntax: "public PrivateEndpointServiceConnectionStatus()"
fields:
- uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus?alt=com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
  desc: "Static value Approved for PrivateEndpointServiceConnectionStatus."
- uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.PENDING"
  fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.PENDING"
  name: "PENDING"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PENDING"
  summary: "Static value Pending for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus?alt=com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus PENDING"
  desc: "Static value Pending for PrivateEndpointServiceConnectionStatus."
- uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.REJECTED"
  fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.REJECTED"
  name: "REJECTED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.REJECTED"
  summary: "Static value Rejected for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus?alt=com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus REJECTED"
  desc: "Static value Rejected for PrivateEndpointServiceConnectionStatus."
methods:
- uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)"
  fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "PrivateEndpointServiceConnectionStatus.fromString(String name)"
  summary: "Creates or finds a Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for"
    name: "name"
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/String.html\">String</a>"
  syntax: "public static PrivateEndpointServiceConnectionStatus fromString(String name)"
  desc: "Creates or finds a PrivateEndpointServiceConnectionStatus from its string representation."
  returns:
    description: "the corresponding PrivateEndpointServiceConnectionStatus"
    type: "<xref href=\"com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus?alt=com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.values()"
  fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.values()"
  name: "values()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.values()"
  modifiers:
  - "static"
  syntax: "public static Collection<PrivateEndpointServiceConnectionStatus> values()"
  returns:
    description: "known PrivateEndpointServiceConnectionStatus values"
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html\">Collection</a>&lt;<xref href=\"com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus?alt=com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
desc: "Defines values for PrivateEndpointServiceConnectionStatus."
metadata: {}
package: "com.microsoft.azure.management.mediaservices.v2020_05_01"
artifact: com.microsoft.azure.mediaservices.v2020_05_01:azure-mgmt-media:1.0.0-beta
