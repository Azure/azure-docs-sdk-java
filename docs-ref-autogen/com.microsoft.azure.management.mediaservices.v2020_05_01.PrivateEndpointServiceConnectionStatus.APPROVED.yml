### YamlMime:JavaMember
uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.microsoft.azure.management.mediaservices.v2020_05_01.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
  desc: "Static value Approved for PrivateEndpointServiceConnectionStatus."
type: "field"
metadata: {}
package: "com.microsoft.azure.management.mediaservices.v2020_05_01"
artifact: com.microsoft.azure.mediaservices.v2020_05_01:azure-mgmt-media:1.0.0-beta
